src/alignment/model_utils.py

```diff
@@ -105,6 +106,9 @@ def is_adapter_model(model_name_or_path: str, revision: str = "main") -> bool:
     except HFValidationError:
         # If not, check local repo
         repo_files = os.listdir(model_name_or_path)
+    except RepositoryNotFoundError:
```

Comment:
If I'm not mistaken we can gather this exception with the one above - would you mind checking that works?

Reply:
Sounds good!  I'll run a quick sanity check

Reply:
@lewtun I added a simple unit test.  If needed I could enhance the test with `unittest.mock`